libs/python/btp_cli.py

```diff
@@ -724,6 +724,13 @@ def create_environments(self):
                         self.create_and_assign_quota_plan(environment)
 
                 elif environment.name == "kymaruntime":
+                    # Support load via dynamic parameter file as well
```

Comment:
Format seems not according to `black` rules,

Can you please run `black .\libs\python\* -t py310`  
See <https://github.com/psf/black>